spec/support/page_objects/trainees/course_details.rb

```diff
@@ -0,0 +1,21 @@
+module PageObjects
+  module Trainees
+    class CourseDetails < PageObjects::BasePage
+      set_url "/trainees/{id}/course-details"
+      element :submit_button, "input[name='commit']"
+      element :course_title, "input[name='trainee[course_title]']"
+      element :course_phase, "input[name='trainee[course_phase]']"
+      element :programme_start_date_day, "input[name='trainee[programme_start_date(3i)]']"
+      element :programme_start_date_month, "input[name='trainee[programme_start_date(2i)]']"
+      element :programme_start_date_year, "input[name='trainee[programme_start_date(1i)]']"
+      element :programme_length, "input[name='trainee[programme_length]']"
+      element :programme_end_date_day, "input[name='trainee[programme_end_date(3i)]']"
+      element :programme_end_date_month, "input[name='trainee[programme_end_date(2i)]']"
+      element :programme_end_date_year, "input[name='trainee[programme_end_date(1i)]']"
+      element :allocation_subject, "input[name='trainee[allocation_subject]']"
+      element :itt_subject, "input[name='trainee[itt_subject]']"
+      element :employing_school, "input[name='trainee[employing_school]']"
+      element :placement_school, "input[name='trainee[placement_school]']"
```

Comment:
Could we not use id selectors instead of `input[type...]` selectors

Reply:
if #24 is merged first then most of this will be done... https://github.com/DFE-Digital/register-trainee-teacher-data/pull/24/files#diff-66b836b7695bdca2ca2a25c8bcb314e7

Reply:
I've rebased from #24 so that's gone 💥